rasr/command.py

```diff
@@ -118,19 +123,53 @@ def select_exe(cls, specific_exe, default_exe_name):
             return cls.default_exe(default_exe_name)
         return specific_exe
 
-    def run_script(self, task_id, log_file, cmd="./run.sh", args=None, retries=2):
+    def run_script(
+        self,
+        task_id: int,
+        log_file: Union[str, Path],
+        cmd: str = "./run.sh",
+        args: List = None,
+        retries: int = 2,
+        use_tmp_dir: bool = False,
+    ):
         args = [] if args is None else args
         tmp_log_file = remove_suffix(
             os.path.basename(tk.uncached_path(log_file)), ".gz"
         )
-        self.run_cmd(cmd, [task_id, tmp_log_file] + args, retries)
+
+        if use_tmp_dir or (
+            hasattr(gs, "JOB_RUN_RASR_TMP_DIR") and gs.JOB_RUN_RASR_TMP_DIR
+        ):
+            with tempfile.TemporaryDirectory(prefix=gs.TMP_PREFIX) as tmp_dir:
```

Comment:
Should this `TMP_PREFIX` be from gs or should this maybe be handled differently? How is it done in other cases where tmp dirs are constructed

Reply:
yes, this is well defined https://github.com/rwth-i6/sisyphus/blob/cc58196b0c410a6d98ba7c1d79ac2cd1eb48ae39/sisyphus/global_settings.py#L273

In other cases we do it similarly. (Mostly because I advocated for it xD)